common/colors.tex

```diff
@@ -0,0 +1,3 @@
+\definecolor{tu}{RGB}{132,182,24}
```

Comment:
```suggestion
\definecolor{tu}{RGB}{132,182,24}
\definecolor{tu-alt}{RGB}{99,154,0}
```
And I would even argue to define `tu` and `tu-alt` the other way around. The cooperate design specifies, that this is the TU-green and the other is for "Microsoft" products. We should follow the cooperate design guide, not what we find preferable. https://itmc.tu-dortmund.de/unsere-services/medien-services/grafik/design/corporate-design/

Reply:
There are two colors on the current TU websites, the second one here is used for buttons and large areas.

According to the TU corporate design document here:
https://service.tu-dortmund.de/documents/18/1022595/130415_corporate_design_manual_TUDortmund.pdf/539f1b75-806a-44fa-8881-503260233ddc

The main color is 

> Pantone 368 C
> CMYK 57/0/100/0
> (RAL 6018)
> HEX #83B818

Converting that hex code to 8-bit integers is 132, 182, 24 and it matches the color inside the TU logo.

I'd use the "logo color".

I can write  a mail to the marketing, but I just think there is a mistake on their site actually...

Reply:
No, I found the pitfall, error on my side: 

>Die Farben im Web unterscheiden sich geringfügig von
den Farben im Print-Corporate-Design. Die Anpassung
der Farbwerte im Web dient der Barrierefreiheit des
Internetauftritts.

Reply:
Thanks for the link

Reply:
Question is: Is this print or is this web? (Nobody will print it, everybody will view it on their screen) Anyway enough of this, I am fine with it. We can define `tu-alt` as `tu-web` so people keep in mind, that there is a seperate web-color...

Reply:
> Die Anpassung
der Farbwerte im Web dient der Barrierefreiheit des
Internetauftritts.

Warum ist ein leicht anderes Grün Barrierefreier???